<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(165, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="botao2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Criptografar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="entrada">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="titulo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Programa de criptografia &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEntrada">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrada de dados -&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Descriptografar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="entrada_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>371</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEntrada_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sa√≠da de dados -&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botao3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>101</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Limpar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
